cmd/utils/flags.go

```diff
@@ -493,6 +493,11 @@ var (
 		Usage: "Sets a cap on gas that can be used in eth_call/estimateGas (0=infinite)",
 		Value: ethconfig.Defaults.RPCGasCap,
 	}
+	RPCGlobalCallTimeoutFlag = cli.DurationFlag{
+		Name:  "rpc.calltimeout",
```

Comment:
```suggestion
		Name:  "rpc.evmtimeout",
```

otherwise it is ambiguous and people could understand it is about rpc calls in general and not eth_call - giving it a more general name so it can later also be used for estimating gas, ..

Reply:
Yep this makes sense. Question below about using in `estimateGas`.